modules/oadp-release-notes-1-1-6.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-6_{context}"]
+= OADP 1.1.6 release notes
+
+The OADP 1.1.6 release notes lists any new features, resolved issues and bugs, and known issues.
+
+[id="new-features1.1.6_{context}"]
+== New features
```

Comment:
is it normal to have empty sections in the OADP release notes? the emptiness looks strange when rendered, so i figure i'd ask

Reply:
**Delete the beggar!!!!**